.gitpod.yml

```diff
@@ -4,17 +4,37 @@ image: gitpod/workspace-postgres
 
 tasks:
   - init: |
-      yarn create redwood-app . --overwrite --yarn-install --typescript --git-init=false
-      node gitpod-setup.js
-  - command: yarn rw dev --fwd="--client-web-socket-url=ws$(gp url 8910 | cut -c 5-)/ws"
+      # Cleanup terminal
+      printf "\033[3J\033c\033[3J"
+
+      # Auto init a redwoodjs project for Gitpod if missing
+      if test ! -e redwood.toml; then {
+        # Create
+        yarn create redwood-app . --overwrite --yarn-install --typescript --git-init=false
+
+        # Custom webpack config for Gitpod
+        yarn rw setup webpack
+        wp_config_file="web/config/webpack.config.js"
+        if ! grep -q "GITPOD_REPO_ROOT" "${wp_config_file}" 2>/dev/null; then {
+          sed -i "s|if (mode === 'development') {|& \n\t\tif (process.env.GITPOD_REPO_ROOT) { config.devServer.client.webSocketURL = require('child_process').execSync(\`gp url \${config.devServer.port}\`).toString().trim().replace('https', 'wss').concat('/ws') }|" "${wp_config_file}"
```

Comment:
This command appends the following code to `web/config/webpack.config.js` file after `if (mode === 'development') {`:
```js
    if (process.env.GITPOD_REPO_ROOT) {
      config.devServer.client.webSocketURL = require('child_process')
        .execSync(`gp url ${config.devServer.port}`)
        .toString()
        .trim()
        .replace('https', 'wss')
        .concat('/ws')
    }
```

This works. But I have a proposal (if appropriate), that is to directly add this piece of code on https://github.com/redwoodjs/redwood/blob/main/packages/cli/src/commands/setup/webpack/templates/webpack.config.js.template, so we won't need this shell command here anymore and Gitpod becomes supported from the source code itself! 😄 

What do you think?